packages/site/src/web/index.js

```diff
@@ -25,6 +25,8 @@ import App from '../components/App';
 import siteTheme from '../components/siteTheme';
 // $FlowFixMe
 import demos from '{{DEMO_LIST_PATH}}';
+require('./index.css');
+require('react-live/react-live.css');
```

Comment:
these get injected into the <head> in <style> tags in the order they appear here. glamorous also injects a <style> tag and I can't figure out how to get glamorous's not to be first...